integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/alerting/ExternalAlerterITest.groovy

```diff
@@ -61,10 +68,12 @@ class ExternalAlerterITest extends AlertingITestBase {
         resp = client.post(path: "triggers", body: triggerTestAvg)
         assertEquals(200, resp.status)
 
-        // ADD external metrics avg condition. Note: alerterId must be "HawkularMetrics"
+        // Note: alerterId must be "HawkularMetrics"
         // Average over the last 1 minute > 50, check every minute
+        Query qNow = Query.builder("qNow").metric(metricId).duration("1mn").build();
```

Comment:
I'm not exactly sure what that test would exercise.  This code is already using REST to post the ExternalCondition.  Are you concerned that python/ruby clients may not be able to post an ExternalCondition using a JSON string for the expression value?